libs/api/repository/src/lib/gn4/gn4-repository.ts

```diff
@@ -190,7 +190,7 @@ export class Gn4Repository implements RecordsRepositoryInterface {
         'bucket',
         null,
         JSON.stringify(
-          this.gn4SearchHelper.getRelatedRecordPayload(similarTo, 3)
+          this.gn4SearchHelper.getRelatedRecordPayload(similarTo, 4)
```

Comment:
Did we define the number to be 4 somewhere? I can't find the info in the ticket, but see pagination in the mockups.

Reply:
I picked 4 like:
![image](https://github.com/user-attachments/assets/9da60d1e-d616-4641-91cd-ff215174738e)

Agreed there is other "related" sections like screenshot below, but not sure if this is the subject of this task, or is it?
![image](https://github.com/user-attachments/assets/9fbf25fc-d1a8-41d9-8093-2711f77059b1)

Reply:
I tests with 10 items, here is the result (i forgot to add the block list)
![image](https://github.com/user-attachments/assets/a94be024-876d-4810-bbcf-9710816ff5c9)

@LHBruneton-C2C is displaying 10 related items here a good option? or can you tell me how many items should we display here please?

Reply:
> @LHBruneton-C2C is displaying 10 related items here a good option? or can you tell me how many items should we display, please?

Thanks for the question @AlitaBernachot. It used to be 3 and it should remain 3, there is no reason to change that.

Reply:
> It used to be 3 and it should remain 3, there is no reason to change that.

So this would be the display for 3 related record (see screenshot below), do you confirm?

![image](https://github.com/user-attachments/assets/dff682d2-6176-4c65-b5ea-6db71ca0fc16)